chicago/views.py

```diff
@@ -332,6 +332,43 @@ def get_context_data(self, **kwargs):
             .order_by("-last_action")[:10]
         )
 
+        attendance = []
+        events = []
+        # get all events across all memberships tenures
```

Comment:
to limit to the current term, you need to get a handle on the dates of the current term. you can do that this way.

```python
from django.utils import timezone
from opencivicdata.legislative.models import LegislativeSession

current_legislative_session = LegislativeSession.objects.get(start_date__lt=timezone.now(), end_date__gt=timezone.now())
```

then you can use the end and start date associated with that session object in your event filter where you are currently using the start and end date of the membership.